examples/extension-server/Makefile

```diff
@@ -6,6 +6,14 @@ tools/controller-gen     = $(tools.bindir)/controller-gen
 $(tools.bindir)/%: $(tools.srcdir)/%/pin.go $(tools.srcdir)/%/go.mod
 	cd $(<D) && GOOS= GOARCH= go build -o $(abspath $@) $$(sed -En 's,^import _ "(.*)".*,\1,p' pin.go)
 
+IMAGE_PREFIX ?= envoyproxy/gateway-
+APP_NAME ?= extension-server
+TAG ?= latest
+
+.PHONY: docker-buildx
+docker-buildx:
```

Comment:
While you're here - unsure if you want to add multi-platform support.

I noticed on a mac arm machine the binary in the image doesn't match the default arch.

You could use `ko` to simplify the build https://ko.build/ - and you won't require docker as a tool here

Reply:
this's just an example, and we won't push it to docker.io, so I won't spend time on supporting multi-platform.

if you're interesting, feel free to send PR in the future.

Reply:
Multi arch scripts love in https://github.com/envoyproxy/gateway/blob/main/tools/make/image.mk in case they need to be perused for extension-server